docs/application/dotnet/guides/nui/tizenshell.md

```diff
@@ -41,3 +41,4 @@ public void TizenShell_INIT()
 ## Related information
 - Dependencies
   - Tizen 6.0 and Higher
+  - Tizen dotnet SDK 1.1.5 and Higer
```

Comment:
Tizen .NET SDK 1.1.5 and Higher

Reply:
Question. 

Is there any reason that we should focus on Tizen .Net SDK version?
If not how about removing this line as the guide pages are focusing on the Tizen version information?

Reply:
I checked examples on Visual Studio Code. There user have to change tizen api version to tizen80, and Tizen .NET SDK to 1.1.5 to use TZSH. If .NET SDK is set to 1.0.3 TZSH is not available.